src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandlerSchema.php

```diff
@@ -38,8 +38,8 @@ private function addSessionTable($tableName)
     {
         $table = $this->createTable($tableName);
         $table->addColumn('sess_id', 'string');
-        $table->addColumn('sess_data', 'text')->setNotNull(true);
-        $table->addColumn('sess_time', 'integer')->setNotNull(true)->setUnsigned(true);
+        $table->addColumn('sess_data', 'text')->setNotnull(true);
```

Comment:
We *must* not, at least not technically (proof: it works without any issue)
Did you try submitting on doctrine?